server.js

```diff
@@ -0,0 +1,3 @@
+const app = require('./app')
+
```

Comment:
Would it be possible to pass all these constants like `'randomWord` and `rules` and `guessRegister` etc into app here instead of instantiating it at the top of the file? That would mean you could possibly pass them in in your tests and set the randomWord etc. 
I'm not sure if it is possible but have you tried?